config/sync/block.block.stanford_basic_branding.yml

```diff
@@ -6,7 +6,7 @@ dependencies:
     - system
   theme:
     - stanford_basic
-id: sitebranding
```

Comment:
I haven't tested this yet but, are we going to need an upgrade path for this or are the old blocks removed on the next config import?

Reply:
no need for an upgrade path. since the UUID's didn't change, it just gets renamed in the DB